FEATURES.md

```diff
@@ -0,0 +1,175 @@
+# Features
+
+This document describes a list of use-cases and features that this library provides.
+
+## Use Cases
+
+The following are the core use cases that this integration supports. If you have another use case, please let us know by creating an Issue.
+
+* UC 1: As a user, I want to trigger a Keptn quality gate evaluation from within Jenkins
+* UC 2: As a user, I want to trigger a delivery with Keptn from within Jenkins
+* UC 3: As a user, I want to create a project, a service, and push files to the Keptn configuration repo
+* UC 4: As a user, I want to trigger an existing CI/CD Jenkins Pipeline from Keptn (via webhook-service) 
+
+
+## Stories
+
+Each use case consists of multipel stories, however, a story can be part of many use cases.
+
+### User Story 1: Install keptn-jenkins-library in Jenkins
+
+**Goal**: A user should be able to install the library within their Jenkins installation.
+
+**DoD**:
+* REAMDE contains installation instructions with screenshots
+* README contains compatibility matrix
+
+
+### User Story 2: Connect to the Keptn API
+
+**Goal**: A user should be able to connect to their existing Keptn installation.
+
+**DoD**:
+* README contains instructions on how to configure secrets / environment variables
+* Shared library code contains helper functions to read secrets / environment variables
+
+
+### User Story 3: Create a project with shipyard
+
+**Goal**: A user should be able to create a project and upload a shipyard file
```

Comment:
Creating a project requires the Shipyard and uploading it afterwards does not really make sense. 

Please change to: `**Goal**: A user should be able to create a project defined by a shipyard file`

Reply:
done